keyboards/ergodox_ez/keymaps/dvorak_42_key/keymap.c

```diff
@@ -55,14 +55,16 @@ enum custom_keycodes {
 };
 
 
-#define BASE         0 // base dvorak layer
-#define KEYNAV       1 // arrow navigation (right hand)
-#define KEYSEL       2 // arrow navigation + shift (allow text selection)
-#define SHELL_NAV    3 // bash shortcuts
-#define SHELL_SCREEN 4 // linux screen shortcuts
-#define SCREEN_NAV   5 // navigate between linux screen tabs 
-#define BROWSER_CONTROL 6 // control browser and mouse
-#define COMBINED      	7 // combined numbers and symbols layer
+#define BASE             0 // base dvorak layer
```

Comment:
You may want to consider using an `enum` here, as it may make organization easier, in the future.